website/docs/docs/collaborate/govern/model-access.md

```diff
@@ -9,6 +9,15 @@ description: "Define model access with group capabilities"
 This functionality is new in v1.5.
 :::
 
+:::info Model access != User access
+
+**Model groups and access** are dbt-core constructs, distinct from **user groups and access** in dbt Cloud. For docs on the latter, see: ["About user access"](about-user-access).
```

Comment:
**Model groups and access** and **user groups and access** mean two different things. "User groups and access" is a specific term used in dbt Cloud. Refer to [User access](/docs/cloud/manage-access/about-user-accessl)  for more info.